components/property_info.rst

```diff
@@ -439,7 +439,18 @@ with the ``property_info`` service in the Symfony Framework::
     $serializerExtractor = new SerializerExtractor($serializerClassMetadataFactory);
 
     // List information.
-    $serializerExtractor->getProperties($class);
+    $serializerExtractor->getProperties($class, ['serializer_groups' => ['mygroup']]);
+    
+.. note::
+
+    The `serializer_groups` option must be provided in order to have a value different than `null` returned.
```

Comment:
A minor thing. I know it's annoying, but RST format is a bit different than Markdown for code. All single backticks must be double backticks:

```
`serializer_groups` -> ``serializer_groups``
```

Reply:
Oh right, thanks :) I've been misoriented by the single backtick for links 😅